designs/plugintest/src/plugin-title.mjs

```diff
@@ -1,9 +1,10 @@
 import { titlePlugin } from '@freesewing/plugin-title'
 import { base } from './base.mjs'
 
-const pluginTitle = ({ points, Point, paths, Path, macro, options, part }) => {
+const pluginTitle = ({ points, Point, paths, Path, macro, options, store, part }) => {
   if (['title', 'all'].indexOf(options.plugin) !== -1) {
-    if (options.titleMeta) part.context.settings.metadata = { for: 'Some user' }
+    if (options.titleMeta) store.data.for = 'Some user'
```

Comment:
Can we use `store.set('data.for', 'Some user')` here?
It's the same under the hood today, but using the method is more future-proof

Reply:
I made the change to `store.set('data.for', 'Some user')`.